File: exercises/beecrowd/1009/1009.go
// https://judge.beecrowd.com/pt/problems/view/1009
package main

import "fmt"

func main() {
	var name string
	var salary, salesAmount float64

	fmt.Scanln(&name)
	fmt.Scanln(&salary)
	fmt.Scanln(&salesAmount)

	total := salary + (salesAmount * 0.15)

	fmt.Printf("TOTAL = R$ %.2f\n", total)
}

File: exercises/beecrowd/1009/1009.ts
// https://judge.beecrowd.com/pt/problems/view/1009
import * as fs from 'fs';

function main(): void {
    const input = fs.readFileSync('/dev/stdin', 'utf8');
    const lines = input.split(/\r?\n/);
    
    const name = String(lines.shift());
    const salary = Number(lines.shift());
    const salesAmount = Number(lines.shift());
    
    const total = salary + (salesAmount * 0.15)
    
    console.log("TOTAL = R$", total.toFixed(2))
}

main()